lekcja10/baza.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <link rel="stylesheet" href="dane.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Makieta</title>
+</head>
+<body>
+<header>
+   <div class="login size">
+        <label>Zalogowany jako: Jakub Poręba</label>
+        <a href="">Wyloguj się</a>
+    </div>
+ <div class="cytat size">
+   <p>"Nie czekaj, aż życie poda ci okazję – stwórz ją sam."</p>
+    </div>
+   </header>
+<main>
+     <div class="documentation size">
+       <a href="">Opis</a>
+       <a href="">Dokumentacja</a>
+       <a href="">Cennik</a>
+       <a href="">pomoc</a>
+       <a href="">Kontakt</a>
+    </div>
+    <div class="size" style="display: flex;  height: calc(100vh - 300px);">
+
+      <div class="second-menu">
+            <a href=""><div>Moduły</div></a>
+            <a href=""><div>Dostep</div></a>
+            <a href=""></a>>Konfiguracja</div></div>
+    <div>
+   <ul style="list-style-type: none;">
+    <li><a href="">Podstawowa</a></li>
+    <li><a href="">Automatyczna</a></li>
+    <li><a href="">Opcje i parametry</a>
+        <ul style="list-style-type: none;">
+            <li><a href="baza.html">Baza danych</a></li>
+            <li><a href="serwer.html">Serwer</a></li>
+            <li><a href="glowny.html">Dostęp zdalny</a></li>
+            <li><a href="wszystkie.html">Wszystkie opcje</a></li>
+        </ul>
+    </li>
+</ul>
+
+         <a href=""></a>Aktualizacje</div></main>
```

Comment:
```suggestion
         <a href="">Aktualizacje</a>
```
Ta linia powinna **prawdopodobnie** miec taka postac